config.js

```diff
@@ -415,7 +371,51 @@ if (argv.botid || argv.bot || argv.id || argv.minrankedhandicap || argv.maxranke
         }
     }
 
-    if (argv.maxrank) {
+    if (argv.minrankranked) {
+        let re = /(\d+)([kdp])/;
+        let results = argv.minrank.toLowerCase().match(re);
+
+        if (results) {
+            if (results[2] == "k") {
+                exports.minrankranked = 30 - parseInt(results[1]);
+            } else if (results[2] == "d") {
+                exports.minrankranked = 30 - 1 + parseInt(results[1]);
+            } else if (results[2] == "p") {
+                exports.minrankranked = 36 + parseInt(results[1]);
+                exports.proonly = true;
+            } else {
+                console.error("Invalid minrankranked " + argv.minrankranked);
+                process.exit();
+            }
+        } else {
+            console.error("Could not parse minrankranked " + argv.minrankranked);
+            process.exit();
+        }
```

Comment:
These lines are duplicate with 397-415, could be made a function.

Reply:
indeed, but i was like let's make this fast

however yes definitely ! will do